pubsub/pubsub_test.go

```diff
@@ -20,7 +18,7 @@ func TestPublish(t *testing.T) {
 
 func TestPublishMultipleListeners(t *testing.T) {
 	key := "TestPublishMultipleListeners"
-	data := MessageStructTest{}
```

Comment:
please also fix this

Reply:
Done in https://github.com/mesg-foundation/core/pull/321/files